site/specs-source/voice.json

```diff
@@ -2660,6 +2660,14 @@
           },
           "machineDetection": {
             "$ref": "#/components/schemas/MachineDetectionConfiguration"
+          },
+          "priority": {
+            "nullable": true,
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 5,
+            "default": 5,
+            "description": "The priority of this call over other calls from your account when outbound call queueing is enabled. A lower value means higher priority, so a priority 1 call takes precedence over a priority 2 call. Range: integer values between 1 - 5. Default value is 5."
```

Comment:
I think there should be a few words here about why you might want this. Like "For instance, if during a call, your application needs to create another outbound call and <Bridge> it with the first call, you would want to make the new call priority 1 so it is placed before other, less time-sensitive calls."

Reply:
Should that go here or should I add it to the page that discusses rate limits and the queueing feature? This is the api spec description.

Reply:
both maybe? Assuming you mean rateLimits.mdx

Reply:
The API spec description ends up generating the documentation for this field though right?

http://bw-api-docs-vapi-971-call-queueing.s3-website-us-east-1.amazonaws.com/apis/voice#operation/createCall

Reply:
It does, it was more a question of whether a field description is the best place for documentation on possible reasons to use the field.

Reply:
Oh I see. Yeah I still think so; uui has a similar situation. Also--call queuing isn't enabled by default so I feel that should be included (machine detection does not do this, so there is not a precedence, but it seems misleading).

Reply:
I very much think it should go here